content/posts/newsletter-013/index.md

```diff
@@ -66,6 +66,23 @@ If needed, a section can be split into subsections with a "------" delimiter.
 
 ## Library & Tooling Updates
 
+### voxel-mapper
+
+![voxel-mapper-img](https://raw.githubusercontent.com/amethyst/voxel-mapper/master/screenshots/blending.png)
+
+[voxel-mapper](https://github.com/amethyst/voxel-mapper) is a new library and
+in-game editor for voxel maps, smooth or cubey. The ultimate goal of the project
+is to make it easy for artists and programmers alike to generate volumetric game
+content, either manually or procedurally. The library's author @bonsairobo has
```

Comment:
Might be nice to add a link to the author here, if they've got a Twitter/blog/etc?